### DISCUSSION TITLE
D Lang setup?

### DISCUSSION THREAD

**Discussion by mjambon** [ahrefs/atd#384] [Q&A]
As an atd project contributor, I need to build and test everything once in a while and in particular when making a release.

In `.circleci/setup-system`, I found that the following invocation allows me to run and pass the atdd tests:
```
$ source $(~/dlang/install.sh ldc -a)
```
I would like to add it to my `.bashrc` but it puts `(ldc-1.35.0)` in front of my shell prompt (defined by the `PS1` variable). I could filter out the `PS1=...` lines from the generated script with `grep -v` but it's a bit hacky.

What's the recommended way of dealing with this?

**Comment by Khady** [comment]
@elrandar can you advise on this one please? Would be nice to have it documented in the repo too.

**Comment by elrandar** [comment]
I looked into it briefly. There does not seem to be any option to disable that in the script. The reason why I'm using that script here is that it pulls the latest stable `ldc2` release, and is also the simplest way to install the compiler.

Getting binaries via a package manager could also be a possibility, and I think picking one version of the compiler to compile for might be the best going forward. I juste know that Debian packages for `ldc` in particular are quite out of date. 

What do you think?
